gce/jupyter-config.yaml

```diff
@@ -0,0 +1,24 @@
+proxy:
+  secretToken: SECRET
+
+singleuser:
+  image:
+    name: daskdev/dask-notebook
+    tag: latest
+  extraEnv:
+    EXTRA_PIP_PACKAGES: gcsfs xarray
```

Comment:
How hard would it be to put xarray-master in here instead? Also, if we can the optional xarray dependencies (scipy/netcdf4/zarr), that will allow us to start testing a few workflows.

Reply:
We can anything that is pip or conda installable.  Are there particular branches that you'd like us to target?

Reply:
I would also be quite happy to walk you through how to manipulate the cluster if you're interested.  Changing this stuff is surprisingly easy.

Reply:
Conda installing scipy/netcdf4 each time is taking a while.  I'm building and pushing new docker images.

Reply:
I've created new docker images and included the files in this PR.  We now have scipy and netCDF4 installed in the image and pip install gcsfs, xarray, and zarr from git master on each image load.